### PR TITLE
feat: add tracing for v1

### PR DESCRIPTION
**Pull request by lucasfernog-crabnebula**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: core/tauri-macros/src/command/wrapper.rs [hunk 1]
```diff
@@ -210,14 +240,22 @@ pub fn wrapper(attributes: TokenStream, item: TokenStream) -> TokenStream {
...
+  let Invoke {
+    id,
+    message,
+    resolver,
+  } = invoke;
+  parse_args(id, function, message, case).map(|args| {
     quote! {
+      use tracing::Instrument;
+
+      let span = tracing::debug_span!("ipc::request::run", id = #id.0);
```
Comment (CrabNejonas) [inline core/tauri-macros/src/command/wrapper.rs:252]:
```suggestion
      let span = tracing::debug_span!("ipc::request::handler", id = #id.0);
```

I would prefer we call this `ipc::request::handler`, to me its more descriptive than `run`

File: core/tauri-macros/src/command/wrapper.rs [hunk 2]
```diff
@@ -242,6 +284,7 @@ fn body_blocking(
   });
 
   Ok(quote! {
+    let _span = tracing::debug_span!("ipc::request::run", id = #id.0).entered();
```
Comment (CrabNejonas) [inline core/tauri-macros/src/command/wrapper.rs:287]:
```suggestion
    let _span = tracing::debug_span!("ipc::request::handler", id = #id.0).entered();
```

same here

**Review by CrabNejonas — APPROVED** [review]

File: core/tauri-macros/src/command/wrapper.rs (hunk 1 above)
Comment (lucasfernog-crabnebula) [reply core/tauri-macros/src/command/wrapper.rs:252]: there's already a ipc::request::handler somewhere else in the code :D

Comment (CrabNejonas) [reply core/tauri-macros/src/command/wrapper.rs:252]: ah yeah just noticed this too, how about `inner` then? idk

Comment (lucasfernog-crabnebula) [reply core/tauri-macros/src/command/wrapper.rs:252]: Maybe call it `handler::run`?
handler means our Tauri IPC handler is running, and `run` means the actual command is running.

**Pull request closed without merging**